### PR TITLE
update permissions.yml added mandrakk citizensteel and blackracerBA

### PR DESCRIPTION
**Pull request by ZargonKing**

### CODE REVIEW

**Comment by Supertre03** [comment]
Remove All Warp Access To Black Racer-BA

**Comment by ZargonKing** [comment]
robo there is a guy called _Voiden_ who is spam killing players at /warp city spawn

File: PermissionsEx/permissions.yml [hunk 1]
```diff
@@ -2912,6 +2912,35 @@ groups:
         - ch.alias.haste5
         - ch.alias.haste6
         - ch.alias.haste7
+   citizensteel:
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: incorrect spacing

Comment (malonnnn) [inline PermissionsEx/permissions.yml]: Commander Steel (also Steel or Citizen Steel) is the name of three superheroes published by DC Comics

Henry Heywood
Hank Heywood III
Nathan Heywood
Nathaniel Heywood (different than nathan)
The New 52

further reading
https://en.wikipedia.org/wiki/Commander_Steel
http://dc.wikia.com/wiki/Nathaniel_Heywood_(New_Earth)
http://dc.wikia.com/wiki/Nathan_Heywood_(Arrow)

File: PermissionsEx/permissions.yml [hunk 2]
```diff
@@ -2912,6 +2912,35 @@ groups:
...
+        - ch.alias.run2
+        - ch.alias.run3
+        - ch.alias.haste0
+        - ch.alias.haste1
+        - ch.alias.haste2
+        - ch.alias.haste3
+        - ch.alias.haste4
+        - ch.alias.haste5
+        - ch.alias.haste6
+        - ch.alias.haste7
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: evidence of super strength, otherwise limit to 5

File: PermissionsEx/permissions.yml [hunk 3]
```diff
@@ -5151,6 +5180,317 @@ groups:
...
+      prefix: '&cMandrakk'
+      suffix: '&c'
+    inheritance:
+    - mortal
+    worlds:
+      speedforce:
+        permissions:
+        - kill.immune
+        - essentials.warps.*
+        - powers.wallrun
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: super speed

File: PermissionsEx/permissions.yml [hunk 4]
```diff
@@ -5151,6 +5180,317 @@ groups:
...
+        - ch.alias.kill
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: killing at by willpower

File: PermissionsEx/permissions.yml [hunk 5]
```diff
@@ -5151,6 +5180,317 @@ groups:
...
+        - ch.alias.mindcontrol
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: mind control

File: PermissionsEx/permissions.yml [hunk 6]
```diff
@@ -5151,6 +5180,317 @@ groups:
...
+        - ch.alias.carryplayer
+        - ch.alias.dismountplayer
+        - ch.alias.stopmindcontrol
+        - ch.alias.modifyspeed
+        - essentials.gamemode.survival
+        - -essentials.protect.entitytarget.bypass
+        - essentials.gamemode.creative
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: matter manipulation

File: PermissionsEx/permissions.yml [hunk 7]
```diff
@@ -5151,6 +5180,317 @@ groups:
...
+        - ch.alias.mandrakk
+        - essentials.speed.fly
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: flying at super speed

File: PermissionsEx/permissions.yml [hunk 8]
```diff
@@ -5151,6 +5180,317 @@ groups:
...
+        - ch.alias.jump0
+        - ch.alias.jump1
+        - ch.alias.jump2
+        - ch.alias.jump3
+        - ch.alias.jump4
+        - ch.alias.jump5
+        - ch.alias.jump6
+        - ch.alias.jump7
+        - ch.alias.jump8
+        - ch.alias.jump9
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: super jump

File: PermissionsEx/permissions.yml [hunk 9]
```diff
@@ -5151,6 +5180,317 @@ groups:
...
+        - ch.alias.speak
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: mind control

File: PermissionsEx/permissions.yml [hunk 10]
```diff
@@ -5151,6 +5180,317 @@ groups:
...
+        - ch.alias.confuse
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: mind control

File: PermissionsEx/permissions.yml [hunk 11]
```diff
@@ -5151,6 +5180,317 @@ groups:
...
+        - ch.alias.unconfuse
+        - ch.alias.intangible
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: intangibility

File: PermissionsEx/permissions.yml [hunk 12]
```diff
@@ -5151,6 +5180,317 @@ groups:
...
+        - ch.alias.tangible
+        - ch.alias.invis
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: invisibility

File: PermissionsEx/permissions.yml [hunk 13]
```diff
@@ -5151,6 +5180,317 @@ groups:
...
+        - ch.alias.vis
+        - libsdisguises.disguise
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: disguising

File: PermissionsEx/permissions.yml [hunk 14]
```diff
@@ -5151,6 +5180,317 @@ groups:
...
+        - ch.alias.run0
+        - ch.alias.run1
+        - ch.alias.run2
+        - ch.alias.run3
+        - ch.alias.run4
+        - ch.alias.run5
+        - ch.alias.run6
+        - ch.alias.run7
+        - ch.alias.run8
+        - ch.alias.run9
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: super speed

File: PermissionsEx/permissions.yml [hunk 15]
```diff
@@ -5151,6 +5180,317 @@ groups:
...
+        - ch.alias.haste1
+        - ch.alias.haste2
+        - ch.alias.haste3
+        - ch.alias.haste4
+        - ch.alias.haste5
+        - ch.alias.haste6
+        - ch.alias.haste7
+        - ch.alias.haste8
+        - ch.alias.haste9
+        - essentials.gamemode
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: matter manipulation

File: PermissionsEx/permissions.yml [hunk 16]
```diff
@@ -5151,6 +5180,317 @@ groups:
...
+        - essentials.tp
+        - essentials.tp.*
+        - essentials.heal
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: healing

File: PermissionsEx/permissions.yml [hunk 17]
```diff
@@ -5151,6 +5180,317 @@ groups:
...
+        - essentials.heal.*
+        - essentials.skull
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: matter manipulation

File: PermissionsEx/permissions.yml [hunk 18]
```diff
@@ -5151,6 +5180,317 @@ groups:
...
+        - essentials.skull.*
+        - ch.alias.goldcarrot
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: matter manipulation

File: PermissionsEx/permissions.yml [hunk 19]
```diff
@@ -5151,6 +5180,317 @@ groups:
...
+        - ch.alias.seeds
+        - ch.alias.wheat
+        - ch.alias.carrot
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: matter manipulation

File: PermissionsEx/permissions.yml [hunk 20]
```diff
@@ -5151,6 +5180,317 @@ groups:
...
+        - ch.alias.meat
+        - ch.alias.fish
+        - ch.alias.bread
+        - ch.alias.wine
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: matter manipulation

File: PermissionsEx/permissions.yml [hunk 21]
```diff
@@ -5151,6 +5180,317 @@ groups:
...
+        - ch.alias.bonemeal
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: matter manipulation

File: PermissionsEx/permissions.yml [hunk 22]
```diff
@@ -5151,6 +5180,317 @@ groups:
...
+        - ch.alias.potato
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: matter manipulation

File: PermissionsEx/permissions.yml [hunk 23]
```diff
@@ -5151,6 +5180,317 @@ groups:
...
+        - ch.alias.carrot
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: matter manipulation

File: PermissionsEx/permissions.yml [hunk 24]
```diff
@@ -5151,6 +5177,317 @@ groups:
...
+      survival:
+        permissions:
+        - kill.immune
+        - -essentials.enchantments.allowunsafe
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: wizard

File: PermissionsEx/permissions.yml [hunk 25]
```diff
@@ -5151,6 +5177,317 @@ groups:
...
+        - essentials.enchantments.*
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: wizard

File: PermissionsEx/permissions.yml [hunk 26]
```diff
@@ -5151,6 +5177,317 @@ groups:
...
+        - essentials.enchant
+        - powers.wallrun
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: super speed

File: PermissionsEx/permissions.yml [hunk 27]
```diff
@@ -5151,6 +5177,317 @@ groups:
...
+        - essentials.warps.*
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: teleportation

File: PermissionsEx/permissions.yml [hunk 28]
```diff
@@ -5151,6 +5177,317 @@ groups:
...
+        - ch.alias.kill
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: killing via willpower

File: PermissionsEx/permissions.yml [hunk 29]
```diff
@@ -5151,6 +5177,317 @@ groups:
...
+        - ch.alias.modifyspeed
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: changing another persons speed

File: PermissionsEx/permissions.yml [hunk 30]
```diff
@@ -5151,6 +5177,317 @@ groups:
...
+        - ch.alias.mindcontrol
+        - ch.alias.carryplayer
+        - ch.alias.dismountplayer
+        - ch.alias.stopmindcontrol
+        - essentials.gamemode.survival
+        - essentials.gamemode.creative
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: matter manipulation

File: PermissionsEx/permissions.yml [hunk 31]
```diff
@@ -5151,6 +5177,317 @@ groups:
...
+        - ch.alias.mandrakk
+        - essentials.feed
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: saturation of others

File: PermissionsEx/permissions.yml [hunk 32]
```diff
@@ -5151,6 +5177,317 @@ groups:
...
+        - -essentials.protect.entitytarget.bypass
+        - essentials.feed.others
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: saturation of others

File: PermissionsEx/permissions.yml [hunk 33]
```diff
@@ -5151,6 +5177,317 @@ groups:
...
+        - powers.pointandclicklightning
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: throwing lightning

**Comment by ZargonKing** [comment]
http://dc.wikia.com/wiki/Monitor_Dax_Novu

powers and abilities listed

File: PermissionsEx/permissions.yml [hunk 34]
```diff
@@ -5151,6 +5177,317 @@ groups:
...
+        - essentials.fly
+        - essentials.speed.fly
+        - ch.alias.touchkill
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: killing via a touch

File: PermissionsEx/permissions.yml [hunk 35]
```diff
@@ -5151,6 +5177,317 @@ groups:
...
+        - ch.alias.jump1
+        - ch.alias.jump2
+        - ch.alias.jump3
+        - ch.alias.jump4
+        - ch.alias.jump5
+        - ch.alias.jump6
+        - ch.alias.jump7
+        - ch.alias.jump8
+        - ch.alias.jump9
+        - ch.alias.speak
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: mind control

File: PermissionsEx/permissions.yml [hunk 36]
```diff
@@ -5151,6 +5177,317 @@ groups:
...
+        - ch.alias.confuse
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: mind control

File: PermissionsEx/permissions.yml [hunk 37]
```diff
@@ -5151,6 +5177,317 @@ groups:
...
+        - ch.alias.unconfuse
+        - ch.alias.intangible
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: intangibility

File: PermissionsEx/permissions.yml [hunk 38]
```diff
@@ -5151,6 +5177,317 @@ groups:
...
+        - ch.alias.tangible
+        - ch.alias.invis
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: invisibility

File: PermissionsEx/permissions.yml [hunk 39]
```diff
@@ -5151,6 +5177,317 @@ groups:
...
+        - ch.alias.vis
+        - libsdisguises.disguise
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: disguising

File: PermissionsEx/permissions.yml [hunk 40]
```diff
@@ -5151,6 +5177,317 @@ groups:
...
+        - ch.alias.run0
+        - ch.alias.run1
+        - ch.alias.run2
+        - ch.alias.run3
+        - ch.alias.run4
+        - ch.alias.run5
+        - ch.alias.run6
+        - ch.alias.run7
+        - ch.alias.run8
+        - ch.alias.run9
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: super speed

File: PermissionsEx/permissions.yml [hunk 41]
```diff
@@ -5151,6 +5177,317 @@ groups:
...
+        - ch.alias.haste2
+        - ch.alias.haste3
+        - ch.alias.haste4
+        - ch.alias.haste5
+        - ch.alias.haste6
+        - ch.alias.haste7
+        - ch.alias.haste8
+        - ch.alias.haste9
+        - essentials.gamemode
+        - essentials.tp
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: teleportation

File: PermissionsEx/permissions.yml [hunk 42]
```diff
@@ -5151,6 +5177,317 @@ groups:
...
+        - essentials.tp.*
+        - essentials.heal
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: healing

File: PermissionsEx/permissions.yml [hunk 43]
```diff
@@ -5151,6 +5177,317 @@ groups:
...
+        - essentials.heal.*
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: healing others

File: PermissionsEx/permissions.yml [hunk 44]
```diff
@@ -5151,6 +5177,317 @@ groups:
...
+        - essentials.repair
+        - essentials.repair.*
+        - essentials.skull
+        - essentials.skull.*
+        - ch.alias.flame
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: controlling/creating fire

File: PermissionsEx/permissions.yml [hunk 45]
```diff
@@ -5151,6 +5177,317 @@ groups:
...
+        - essentials.time
+        - essentials.time.*
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: modifying time

File: PermissionsEx/permissions.yml [hunk 46]
```diff
@@ -5151,6 +5177,317 @@ groups:
...
+        - essentials.spawner
+        - essentials.spawner.*
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: matter manipulation

File: PermissionsEx/permissions.yml [hunk 47]
```diff
@@ -5151,6 +5177,317 @@ groups:
...
+        - essentials.lightning
+        - essentials.lightning.others
+        - essentials.thunder
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: muting thunder

File: PermissionsEx/permissions.yml [hunk 48]
```diff
@@ -5151,6 +5177,317 @@ groups:
...
+        - essentials.weather
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: changing weather

File: PermissionsEx/permissions.yml [hunk 49]
```diff
@@ -5151,6 +5177,317 @@ groups:
...
+        - essentials.back
+        - essentials.back.*
+        - essentials.top
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: teleportation

File: PermissionsEx/permissions.yml [hunk 50]
```diff
@@ -5151,6 +5177,317 @@ groups:
...
+        - ch.alias.goldcarrot
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: matter manipulation

File: PermissionsEx/permissions.yml [hunk 51]
```diff
@@ -5151,6 +5177,317 @@ groups:
...
+        - ch.alias.seeds
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: matter manipulation

File: PermissionsEx/permissions.yml [hunk 52]
```diff
@@ -5151,6 +5177,317 @@ groups:
...
+        - ch.alias.wheat
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: matter manipulation

File: PermissionsEx/permissions.yml [hunk 53]
```diff
@@ -5151,6 +5177,317 @@ groups:
...
+        - ch.alias.carrot
+        - ch.alias.meat
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: matter manipulation

File: PermissionsEx/permissions.yml [hunk 54]
```diff
@@ -5151,6 +5177,317 @@ groups:
...
+        - ch.alias.fish
+        - ch.alias.bread
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: matter manipulation

File: PermissionsEx/permissions.yml [hunk 55]
```diff
@@ -5151,6 +5177,317 @@ groups:
...
+        - ch.alias.wine
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: matter manipulation

File: PermissionsEx/permissions.yml [hunk 56]
```diff
@@ -5151,6 +5177,317 @@ groups:
...
+        - ch.alias.bonemeal
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: matter manipulation

File: PermissionsEx/permissions.yml [hunk 57]
```diff
@@ -5151,6 +5177,317 @@ groups:
...
+        - ch.alias.potato
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: matter manipulation

File: PermissionsEx/permissions.yml [hunk 58]
```diff
@@ -5151,6 +5177,317 @@ groups:
...
+        - ch.alias.carrot
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: matter manipulation

File: PermissionsEx/permissions.yml [hunk 59]
```diff
@@ -5151,6 +5180,317 @@ groups:
...
+    options:
+      prefix: '&cMandrakk'
+      suffix: '&c'
+    inheritance:
+    - mortal
+    worlds:
+      speedforce:
+        permissions:
+        - kill.immune
+        - essentials.warps.*
```
Comment (ZargonKing) [inline PermissionsEx/permissions.yml]: http://3.bp.blogspot.com/--pgTjCEsaZM/UQ01ZzR95rI/AAAAAAAALHA/8U0aLqJh_SA/s640/flash16c+cant+escape+speed+force+iris.jpg

File: PermissionsEx/permissions.yml (hunk 3 above)
Comment (ZargonKing) [inline PermissionsEx/permissions.yml]: http://static4.comicvine.com/uploads/square_small/11114/111147698/4829011-1443598328218.jpg

File: PermissionsEx/permissions.yml (hunk 59 above)
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: that's not black racer ?

File: PermissionsEx/permissions.yml (hunk 3 above)
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: im retarded sorry this was a stupid one

**Comment by malonnnn** [comment]
these heroes should be broken apart into their own pull requests

**Comment by malonnnn** [comment]
closed because someone else already did mandrakk

**Pull request closed without merging**